docs/docs/dev/security/certificates.md

```diff
@@ -28,102 +27,89 @@ Note, this CA is not the same CA for service-to-service communication.
 
 **To override autogenerated certificates**
 
-:::: tabs :options="{ useUrlFragment: false }"
-::: tab "Kubernetes (kumactl)"
+::: tip
+If overridden, Kuma is using the certificates to protect not only [data plane proxy to control plane](#data-plane-proxy-to-control-plane-communication) traffic
+but also [user to control plane](#user-to-control-plane-communication) traffic
+and [control plane to control plane](#control-plane-to-control-plane-multizone) traffic.
+:::
```

Comment:
Let's change `KUMA_DP_SERVER_TLS_CERT_FILE` to `KUMA_GENERAL_TLS_CERT_FILE` then in Universal tab